app/src/main/java/org/thoughtcrime/securesms/groups/compose/Components.kt

```diff
@@ -0,0 +1,169 @@
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import network.loki.messenger.R
+import network.loki.messenger.libsession_util.util.Contact
+import org.session.libsession.utilities.Address
+import org.session.libsession.utilities.recipients.Recipient
+import org.session.libsignal.utilities.AccountId
+import org.thoughtcrime.securesms.groups.ContactItem
+import org.thoughtcrime.securesms.ui.Avatar
+import org.thoughtcrime.securesms.ui.components.RadioButton
+import org.thoughtcrime.securesms.ui.theme.LocalColors
+import org.thoughtcrime.securesms.ui.theme.LocalDimensions
+import org.thoughtcrime.securesms.ui.theme.LocalType
+import org.thoughtcrime.securesms.ui.theme.PreviewTheme
+
+
+@Composable
+fun GroupMinimumVersionBanner(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(LocalColors.current.warning)
+    ) {
+        Text(
+            text = stringResource(R.string.groupInviteVersion),
+            color = LocalColors.current.textAlert,
+            style = LocalType.current.small,
+            maxLines = 2,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.padding(horizontal = 24.dp, vertical = 4.dp)
+        )
+    }
+}
+
+fun LazyListScope.multiSelectMemberList(
+    contacts: List<ContactItem>,
+    modifier: Modifier = Modifier,
+    onContactItemClicked: (accountId: AccountId) -> Unit,
+    enabled: Boolean = true,
+) {
+    items(contacts.size) { index ->
+        val contact = contacts[index]
+        Column(modifier = modifier) {
+            if (index == 0) {
+                // Show top divider for the first item only
+                HorizontalDivider(color = LocalColors.current.borders)
+            }
+
+            RadioButton(
+                onClick = { onContactItemClicked(contact.accountID) },
+                selected = contact.selected,
+                enabled = enabled,
+                contentPadding = PaddingValues(
+                    vertical = LocalDimensions.current.xxsSpacing,
+                    horizontal = LocalDimensions.current.smallSpacing
+                )
+            ) {
+                ContactPhoto(contact.accountID)
+                Spacer(modifier = Modifier.size(LocalDimensions.current.smallSpacing))
+                MemberName(name = contact.name)
+            }
+
+            HorizontalDivider(color = LocalColors.current.borders)
+        }
+    }
+}
+
+@Composable
+fun RowScope.MemberName(
+    name: String,
+    modifier: Modifier = Modifier
+) = Text(
+    text = name,
+    style = LocalType.current.h8,
+    color = LocalColors.current.text,
+    modifier = modifier
+        .weight(1f)
+        .align(CenterVertically)
+)
+
+
+@Composable
+fun ContactPhoto(sessionId: AccountId) {
+    return if (LocalInspectionMode.current) {
```

Comment:
If would be good to move this into the Avatar class so that everyone can benefit from the "preview". Might be worth adding a comment to explain what it does too.
Also the `Avatar` call below can be made with an `Address` instead of a recipient if that helps

Reply:
Done